src/Microsoft.ApplicationInsights.AspNetCore/Implementation/TelemetryConfigurationOptionsSetup.cs

```diff
@@ -48,6 +52,22 @@ public void Configure(TelemetryConfiguration configuration)
                 configuration.InstrumentationKey = this.applicationInsightsServiceOptions.InstrumentationKey;
             }
 
+            if (this.telemetryModuleConfigurators.Any())
+            {
+                foreach (ITelemetryModuleConfigurator telemetryModuleConfigurator in this.telemetryModuleConfigurators)
+                {                    
+                    ITelemetryModule telemetryModule = this.modules.FirstOrDefault(((module) => module.GetType() == telemetryModuleConfigurator.GetTelemetryModuleType()));
+                    if (telemetryModule != null)
+                    {
+                        telemetryModuleConfigurator.Configure(telemetryModule);
+                    }
+                    else
+                    {
+                        AspNetCoreEventSource.Instance.LogWarningMessage("Module not found in service collection. Hence will not be configured.");
```

Comment:
It would be helpful to log the type name that was tried and not found.

Reply:
done